src/Select.js

```diff
@@ -284,6 +290,7 @@ var Select = React.createClass({
 
 	selectValue: function(value) {
 		if (!this.props.multi) {
+			this._blurAfterUpdate = (value != 0);
```

Comment:
Why `!= 0`? I thought all the values were strings (at least we get back a comma separated list of strings when we listen to onChange)